app/src/main/java/org/systers/mentorship/viewmodels/CommentsViewModel.kt

```diff
@@ -0,0 +1,114 @@
+package org.systers.mentorship.viewmodels
+
+import android.annotation.SuppressLint
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.observers.DisposableObserver
+import io.reactivex.schedulers.Schedulers
+import org.systers.mentorship.MentorshipApplication
+import org.systers.mentorship.R
+import org.systers.mentorship.models.Comments
+import org.systers.mentorship.models.Task
+import org.systers.mentorship.models.User
+import org.systers.mentorship.remote.datamanager.CommentDataManager
+import org.systers.mentorship.remote.datamanager.TaskDataManager
+import org.systers.mentorship.remote.requests.CreateComment
+import org.systers.mentorship.remote.requests.CreateTask
+import org.systers.mentorship.remote.responses.CustomResponse
+import org.systers.mentorship.utils.CommonUtils
+import retrofit2.HttpException
+import java.io.IOException
+import java.util.concurrent.TimeoutException
+
+class CommentsViewModel: ViewModel() {
+
+    var tag = CommentsViewModel::class.java.simpleName!!
+
+    lateinit var commentsList: List<Comments>
+
+    private val commentsDataManager: CommentDataManager = CommentDataManager()
+    val successfulGet: MutableLiveData<Boolean> = MutableLiveData()
+    val successfulAdd: MutableLiveData<Boolean> = MutableLiveData()
```

Comment:
Not the recommended way since the value of mutableLiveData can be changed. Make this private and use another live data object instead.